styleguide/source/assets/scss/03-molecules/_tools-learn-more.scss

```diff
@@ -0,0 +1,11 @@
+.ama__tools--learn-more {
```

Comment:
learn-more is a sub element of tools and should have a double underline to denote that.